projects/aws/eks-a-admin-image/provisioners/install_eksa.sh

```diff
@@ -31,7 +31,7 @@ rm -rf eksctl_tmp
 
 # Install eksctl-anywhere
 mkdir eksa_tmp
-if [[ "$EKSA_VERSION" == "latest" ]]; then
+if [[ "$EKSA_VERSION" == "latest" || "$EKSA_VERSION" == "v0.0.0" ]]; then
```

Comment:
curious about this, why is it needed?

Reply:
I did it so default version is a version number rather than latest which gets added the template name.